ACE/include/makeinclude/platform_android.GNU

```diff
@@ -9,7 +9,123 @@ no_hidden_visibility ?= 1
 # as of NDK r6 inlining is required
 inline ?= 1
 
-include $(ACE_ROOT)/include/makeinclude/platform_linux_common.GNU
+# start of: include most of platform_linux_common.GNU
```

Comment:
Fair enough. I added this for my project but wasn't sure if it was needed in the master repo. I'll remove it in my next update when this round of the review is complete.